lib/sqlalchemy/util/langhelpers.py

```diff
@@ -1737,3 +1737,17 @@ def inject_param_text(doctext, inject_params):
         lines.append(line)
 
     return "\n".join(lines)
+
+
+def _repr_tuple_names(names):
```

Comment:
I think it may be better without the leading `_`, since the modules inside the utils packages are considered private anyway.

Reply:
Hi @CaselIT, 
Noted thanks. I will make this change and i just noted a typo i made in the code in regards to the number of characters which i will amend also.

Kind Regards,
Ramon